packages/angular/src/builders/index.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ *
+ * @fileoverview Bazel builder
+ */
+
+import {BuilderContext, BuilderOutput, createBuilder} from '@angular-devkit/architect';
+import {JsonObject} from '@angular-devkit/core';
+import {getNativeBinary as bazeliskBin} from '@bazel/bazelisk/bazelisk';
+import {getNativeBinary as ibazelBin} from '@bazel/ibazel';
+import {spawn} from 'child_process';
+import {Schema} from './schema';
+
+function _bazelBuilder(
```

Comment:
This can be an `async` function, and instead of using `return Promise.resolve({success: false});` you can simple `return {success: false}`.

Reply:
Ah, I see, I can still return a Promise in the case where I need to nest the resolution of the promise inside the ps.on callback. See how often I do async programming lately...